vitrina/datasets/forms.py

```diff
@@ -64,6 +65,8 @@ def __init__(self, *args, **kwargs):
             Submit('submit', button, css_class='button is-primary')
         )
 
+        self.fields['category'].choices = MoveNodeForm.mk_dropdown_tree(Category)
```

Comment:
It would be nice to get rid of `-- root --` option and instead show `---------` as in other select fields, to indicate empty value.